test/spec/ol/format/wfsformat.test.js

```diff
@@ -811,6 +839,57 @@ describe('ol.format.WFS', function() {
 
   });
 
+  describe('when parsing multiple feature types (MapServer)', function() {
+
+    var features;
+    before(function(done) {
+      afterLoadText('spec/ol/format/gml/multiple-typenames-mapserver.xml', function(xml) {
+        try {
+          features = new ol.format.WFS().readFeatures(xml);
+        } catch (e) {
+          done(e);
+        }
+        done();
+      });
+    });
+
+    it('reads all features', function() {
+      expect(features.length).to.be(3);
```

Comment:
I just realized that this test is wrong. If you don't give a featureType you should get the features for all types (5 in this example). That is another issue.